src/app/components/ledger/ledgerAddressComponent/index.tsx

```diff
@@ -2,7 +2,7 @@ import styled from 'styled-components';
 import { Tooltip } from 'react-tooltip';
 import { useState } from 'react';
 
-import CopySVG from '@assets/img/Copy.svg';
+import copyIcon from '@assets/img/Copy.svg';
```

Comment:
FYI we are in the middle of migrating to phosphor icon set, see here for usage:
https://github.com/secretkeylabs/xverse-web-extension/blob/069859a25e37c2dbbaabc93a151be1dcc45189fc/src/app/components/seedPhraseInput/index.tsx#L69

Reply:
Thanks for the heads up!